### PR TITLE
Add Dataverse Space

### PR DESCRIPTION
**Pull request by Hwesta**
- Transfer Source only
- Space.path should be empty
- Location.path is a search query for Dataverse
- Saves the dataset information as `dataset.json`
- Reads the `dataset.json` to fetch all the files in the dataset
  *\* Tabfiles are the access copies of the original, so fetch the original as a bundle
- Dataverse Agent info is written as an `agents.json`

### CODE REVIEW

File: storage_service/locations/models/dataverse.py [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+            'entries': entries,
+            'properties': properties,
+        }
+
+    def move_to_storage_service(self, src_path, dest_path, dest_space):
+        """
+        Fetch dataset with ID `src_path` to dest_space.staging_path/dest_path.
+        """
+        # TODO how to strip location path if location isn't passed in?
+        # HACK strip everything that isn't a number
```
Comment (mistydemeo) [inline storage_service/locations/models/dataverse.py:103]: What do the location paths look like?

**Comment by mistydemeo** [comment]
:+1:

File: storage_service/locations/models/dataverse.py (hunk 1 above)
Comment (Hwesta) [inline storage_service/locations/models/dataverse.py:103]: Location path is the search query for Dataverse. Eg `*` or `Part of Title`

**Comment by Hwesta** [comment]
Added a warning to the Space creation page that Dataverse is in beta

**Pull request merged**